libraries/AP_HAL_ESP32/boards/esp32empty.h

```diff
@@ -145,5 +145,7 @@
 #define AP_RCPROTOCOL_ENABLED 0
 
 // disable all frames for sim on hw except quad to save DRAM .bss
+#define AP_MOTORSMULTI_BACKEND_DEFAULT_ENABLED 0
+#define AP_MOTORS_MATRIX_ENABLED 1
```

Comment:
Is this comment wrong now?

Reply:
I don't think so - this just allows us to remove even more code on esp32 (coax and single, I believe, maybe tri?)